packages/crypto-consensus-bls12-381/source/schemas.ts

```diff
@@ -1,8 +1,13 @@
 import { schemas as originalSchemas } from "@mainsail/crypto-key-pair-bls12-381";
 
 export const schemas = {
-	publicKey: {
+	consensusPublicKey: {
 		...originalSchemas.publicKey,
 		$id: "consensusPublicKey",
 	},
+	consensusSignature: {
+		$id: "consensusSignature",
+		allOf: [{ $ref: "alphanumeric" }, { maxLength: 192, minLength: 192 }],
```

Comment:
Can be merged with original schema.

Reply:
Signatures don't seem to have any schemas